plugin/vim-npr.vim

```diff
@@ -23,7 +23,7 @@ if !exists("g:vim_npr_default_dirs")
   let g:vim_npr_default_dirs = ["src", "lib", "test", "public", "node_modules"]
 endif
 
-function! VimNPRFindFile(cmd) abort
+function! VimNPRFindFile(cmd, place) abort
```

Comment:
In order to maintain backwards compatibility, may I suggest that we don't modify the signature of this function, but instead add a new one which this and the two others will call. This will prevent breaking the library for people who are already using this function in their configs.

Maybe something along the lines of the pseudocode below.
```
VimNPRFindFile(cmd):
  s:FindFile(cmd, place)

VimNPRFindFile_NewWindow:
  s:FindFile(cmd, place)

VimNRPFindFile_NewTab:
  s:FindFile(cmd, place)

s:FindFile(cmd, place):
  ...
```